samples/SimpleDataSource/SimpleCustomDataProcessor.cs

```diff
@@ -17,22 +17,22 @@ namespace SampleCustomDataSource
     //
     // This is a sample Custom Data Processor that processes simple text files.
     //
-    // Custom Data Processors are created in Custom Data Sources and are used to actually process the file(s).
-    // An instance of this Processor is created for each set of files opened whereas only one instance of the Custom Data Source is ever created.
-    // Note that CustomDataProcessorBase does not require any file(s) in its constructor, so another
-    // implementation might only store/process one file per instance.
+    // Custom Data Processors are created in Processing Source and are used to actually process the source(s).
```

Comment:
nit: plurality doesn't match. either use "created by a Processing Source" or "created in Processing Sources"